apps/changelog-generator/src/Cli.ts

```diff
@@ -1,8 +1,108 @@
-import * as Command from '@effect/cli/Command';
+import { Command, Options } from '@effect/cli';
+import type { ConfigType, DeploymentConfigV2 } from '@blocksense/config-types';
+import {
+  configDir,
+  readConfig,
+  readEvmDeployment,
+} from '@blocksense/config-types';
+import { Console, Effect } from 'effect';
+import Table from 'cli-table3';
+import type { NetworkName } from '@blocksense/base-utils';
+import { parseNetworkName } from '@blocksense/base-utils';
+import { Terminal } from '@effect/platform';
 
-const command = Command.make('hello');
+const listFeeds = Command.make(
+  'list-feeds',
+  {
+    dir: Options.directory('dir').pipe(Options.withDefault(configDir)),
+  },
+  ({ dir }) =>
+    Effect.gen(function* (_) {
+      const network: NetworkName = yield* _(getNetwork);
+      const config = yield* _(
```

Comment:
Is wrapping `Effect.tryPromise(..)` in `_(..)` necessary? Can you try if you can do 
`yield* Effect.tryPromise(..)`?